src/crypto.rs

```diff
@@ -48,15 +47,15 @@ mod tests {
         let seed =
             b"0001020304050607000102030405060700010203040506070001020304050607";
         let pwd = blake3::hash("greatpassword".as_bytes());
+        let pwd = pwd.as_bytes();
```

Comment:
If you remove the blake3 from tests, you can then remove blake3 from the library dependencies (and sha too)